.travis.yml

```diff
@@ -1,42 +1,32 @@
 language: python
 sudo: false
-cache:
-- pip
+dist: trusty
+cache: pip
 python:
-- '2.7'
-- '3.4'
-- '3.5'
-- pypy
```

Comment:
Do we want to drop PyPy support? It is a very popular distribution and just received a lot more funding, so I think keeping it around wouldn't really hurt anyone.